Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN --mount=type=bind,source=.,target=.,readonly\
 
 
 FROM debian:bookworm-slim AS wardend
+RUN apt update && apt install ca-certificates
```

Comment:
Should we add?    && rm -rf /var/lib/apt/lists/*

Reply:
added